src/TvMaze.Api.Client/Configuration/RetryRateLimitingStrategy.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Polly;
+
+namespace TvMaze.Api.Client.Configuration
+{
+    /// <summary>
+    /// This implementation will retry the request multiple times, if a rate limit is encountered.
+    /// </summary>
+    public class RetryRateLimitingStrategy : IRateLimitingStrategy
+    {
+        private readonly AsyncPolicy<HttpResponseMessage> _policy;
+
+        /// <summary>
+        /// Creates an instance with reasonable default values for the rate limits described by the API.
+        /// </summary>
+        public RetryRateLimitingStrategy()
+            : this(5, TimeSpan.FromSeconds(3))
+        {
+        }
+
+        /// <summary>
+        /// Creates an instance that uses the provided configuration values for retrying.
+        /// </summary>
+        /// <param name="retries"></param>
+        /// <param name="retryInterval"></param>
+        public RetryRateLimitingStrategy(int retries, TimeSpan retryInterval)
+        {
+            _policy = Policy
+                .HandleResult<HttpResponseMessage>(message => message.StatusCode == (HttpStatusCode)429)
```

Comment:
Can you move the response code to a constant as well? Unfortunately the `HttpStatusCode.TooManyRequests` enum is only available on `netstandard2.1`